src/components/calculator.js

```diff
@@ -1,42 +1,60 @@
+      <div className="calculator-container">
+        <div className="display-area">
+          <p className="display-results">
+            { total }
+            { operation }
+            { next }
+          </p>
+        </div>
+        <div className="buttons-container">
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, 'AC')); }}>AC</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '+/-')); }}>+/-</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '%')); }}>%</button>
+          <button type="button" className="operator-button" onClick={() => { this.onClickHandler(calculate(this.state, '÷')); }}>÷</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '7')); }}>7</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '8')); }}>8</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '9')); }}>9</button>
+          <button type="button" className="operator-button" onClick={() => { this.onClickHandler(calculate(this.state, 'x')); }}>x</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '4')); }}>4</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '5')); }}>5</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '6')); }}>6</button>
+          <button type="button" className="operator-button" onClick={() => { this.onClickHandler(calculate(this.state, '-')); }}>-</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '1')); }}>1</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '2')); }}>2</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '3')); }}>3</button>
+          <button type="button" className="operator-button" onClick={() => { this.onClickHandler(calculate(this.state, '+')); }}>+</button>
+          <button type="button" className="calculator-button zero" onClick={() => { this.onClickHandler(calculate(this.state, '0')); }}>0</button>
+          <button type="button" className="calculator-button" onClick={() => { this.onClickHandler(calculate(this.state, '.')); }}>.</button>
+          <button type="button" className="operator-button" onClick={() => { this.onClickHandler(calculate(this.state, '=')); }}>=</button>
```

Comment:
- [ ] When a number is divided by zero, it returns an error on the browser. Kindly implement an error handling condition. 
![Screenshot from 2021-08-20 03-00-13](https://user-images.githubusercontent.com/64894927/130143384-1c511b08-47d1-401e-9453-35bbcdc7fa75.png)